app/test_app/functional/api/test_user_api.py

```diff
@@ -34,18 +33,18 @@ def submit(*args, **kwargs):
     return fn_inputs
 
 
-@pytest.mark.parametrize("user_phone_accessor, phone, use_card, business_usage_name, referred_by, "
+@pytest.mark.parametrize("user_phone_accessor, phone, use_card, business_usage_name, gps_location, referred_by, "
                          "initial_disbursement, tier, status_code",
                          [
-                             (lambda o: o.phone, None, False, 'Fuel/Energy', '+61401391419', None, 'superadmin', 400),
-                             (lambda o: o.phone, None, True, 'Fuel/Energy', '+61401391419', None, 'superadmin', 200),
-                             (lambda o: o.phone, fake.msisdn(), False, 'Fuel/Energy', fake.msisdn(), 0, 'superadmin', 200),
-                             (lambda o: o.phone, fake.msisdn(), False, 'Fuel/Energy', fake.msisdn(), None, 'superadmin', 200),
-                             (lambda o: o.phone, fake.msisdn(), False, 'Food/Water', fake.msisdn(), None, 'view', 403)
+                             (lambda o: o.phone, None, False, 'Fuel/Energy', None, '+61401391419', None, 'superadmin', 400),
+                             (lambda o: o.phone, None, True, 'Fuel/Energy', '12.02 -15.04 otheruselessdata!##', '+61401391419', None, 'superadmin', 200),
```

Comment:
could be good to add a test that has invalid lat lng coords... i.e. `12.01 banana 12.1`

Reply:
there's one like that in the unit test :)